make-webpack-config.js

```diff
@@ -25,6 +26,7 @@ module.exports = function(options){
   }
 
   return {
+    devtool: isProd ? 'hidden-source-map' : 'eval',
```

Comment:
@FredyC Thanks for the catch. I think `source-map` is probably best, quoting from the docs:

> devtool: "source-map" cannot cache SourceMaps for modules and need to regenerate complete SourceMap for the chunk. It’s something for production.